src/libs/constants/fonts.ts

```diff
@@ -1,4 +1,5 @@
 const fontSizes = {
+	extraLarge: "36px",
```

Comment:
update theme -> fontSize too

Reply:
dont see update

Reply:
we use  these constants with colors, fonts sizes and weights to define our colors and fonts in theme. I've added it to theme and in code use only from theme.